src/main/steps/common/equality/get.ts

```diff
@@ -0,0 +1,109 @@
+import { Logger } from '@hmcts/nodejs-logging';
```

Comment:
this shd go into a method inside controller file and need to be invoked from CYA